server/controllers/tables.js

```diff
@@ -13,8 +13,9 @@ const getAllTables = async (req, res) => {
 const validateTable = async (req, res) => {
     const name = req.body.name;
     const password = req.body.password
+    const username = req.body.username;
 
-    const table = await tableService.validateTable(name, password);
+    const table = await tableService.validateTable(name, password, username);
```

Comment:
why need also uername ?

Reply:
to add him to the playersOnTable (Database) after the server check the valid of that table.